css/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+		width: calc(100vw - 20px);
+		box-sizing: border-box;
+		padding: 20px;
+		font-size: .8rem;
```

Comment:
where is declaration for font-size in  root?

Reply:
What do you exactly mean by "root"?